pontoon/api/tests/test_urls.py

```diff
@@ -0,0 +1,72 @@
+import sys
+import unittest
+
+from django.test import TestCase
+from django.conf import settings
+from django.core.urlresolvers import clear_url_caches
+
+
+def reload_urls():
+    clear_url_caches()
+    reload(sys.modules[settings.ROOT_URLCONF])
+
+
+class TestGraphiQL(TestCase):
+
+    body = {
+        'query': '{projects{name}}'
+    }
+
+    def test_graphql_dev_get(self):
+        with self.settings(DEV=True):
+            response = self.client.get('/graphql', self.body,
+                                       HTTP_ACCEPT="application/json")
```

Comment:
I don't like this styling. It is harder to maintain, as if you change the variable name for example, you have to realign the entire block. Instead, can you write these like that: 

```python
response = self.client.get(
    '/graphql', 
    self.body,
    HTTP_ACCEPT="application/json"
)
```

Also, remember that we allow lines to be 100 characters long nowadays. Maybe that one can fit on a single line?